settings.gradle

```diff
@@ -52,16 +52,17 @@ include ':jgiven-core',
 
 def release = System.env.RELEASE == "true"
 def android = System.env.ANDROID == "true"
+boolean hasAndroidSourcePath = ( System.env.ANDROID_SDK_ROOT )
 
 if (JavaVersion.current().isJava9Compatible()) {
     include ':jgiven-java9-tests'
 }
 
-if (android || release) {
+if ((android || release) && hasAndroidSourcePath) {
```

Comment:
Is this excluded for developers which have no Android SDK?

Reply:
err yes. After all if you attempt to build everything but don't have that SDK path set, then the android build will throw a beatiful error